src/libraries/System.Security.Cryptography.Xml/tests/SignatureSupport.cs

```diff
@@ -0,0 +1,20 @@
+namespace System.Security.Cryptography.Xml.Tests
```

Comment:
License header is missing.

```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.

namespace System.Security.Cryptography.Xml.Tests
```

Reply:
(I thought we had a lint for that)